Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocRowRenderer.cs

```diff
@@ -188,6 +205,48 @@ private IBocIndexColumnRenderer GetIndexColumnRenderer ()
       return _indexColumnRenderer;
     }
 
+    private void RenderValidationRow (
+        BocListRenderingContext renderingContext,
+        BocListRowRenderingContext rowRenderingContext,
+        BocListDataRowRenderEventArgs rowRenderEventArgs,
+        IReadOnlyCollection<BocListValidationFailureWithLocationInformation> validationFailuresWithLocationInfo,
+        int rowIndex)
+    {
+      var cssClassValidationRow = GetCssClassValidationRow(rowRenderingContext.IsSelected, rowRenderEventArgs);
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, cssClassValidationRow);
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Tr);
+
+      var columnRenderers = renderingContext.ColumnRenderers;
+
+      var lastColumnIndex = 0;
+      for (int i = columnRenderers.Length - 1; i >= 0; i--)
+      {
+        if (lastColumnIndex > columnRenderers[i].VisibleColumnIndex)
+          continue;
+
+        lastColumnIndex = columnRenderers[i].VisibleColumnIndex + 1;
+        break;
+      }
```

Comment:
I try to never have `continue` in a loop and to avoid `break` to make the code easier to reason over. Having both `continue` and `break` in a single loop makes me queasy. I did not yet think how to improve this, am open for a pair session on this. One final break could be okay if we need to have an early exit condition.